.github/workflows/build-v3.yml

```diff
@@ -0,0 +1,87 @@
+name: build v3
+
+on:
+  pull_request:
+    branches: ['main']
+  push:
+    branches: ['main']
+    paths-ignore:
+      - 'docs/**'
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
```

Comment:
I assume you copied from the build.yml.
[req] Please also do so for dependabot step [here](https://github.com/SAP/cloud-sdk-js/blob/v2-main/.github/workflows/build.yml#L98).